ember-cli-build.js

```diff
@@ -29,5 +29,11 @@ module.exports = function(defaults) {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
+  app.import('bower_components/blueimp-file-upload/js/vendor/jquery.ui.widget.js');
+  app.import('bower_components/blueimp-file-upload/js/jquery.iframe-transport.js');
+  app.import('bower_components/cloudinary/js/load-image.min.js');
+  app.import('bower_components/cloudinary/js/canvas-to-blob.min.js');
+  app.import('bower_components/cloudinary/js/jquery.cloudinary.js');
+
```

Comment:
@swati I see, in that case I think we only need `canvas-to-blob.js`. The blueimp-file-upload will be to do with file uploading of course, and apparently [load-image.js](https://github.com/blueimp/JavaScript-Load-Image) allows you to display images from file upload controls, so don't think we need it.